### DISCUSSION TITLE
Turn off Stream Deck screen while computer displays are off

### DISCUSSION THREAD

**Discussion by pflavio** [StreamController/StreamController#293] [General]
Hey! I'm looking for a way to turn off the Stream Deck screen while my computer displays are off (but the computer itself is powered on, NOT in suspend). I can turn off all RGB lighting on the computer and the attached devices, both screens but the only thing that stays on in the Stream Deck. Is there something like a power saving state or similar? Thanks.

**Comment by jfbauer432** [comment]
I don't know if the Stream Decks have that ability, but there may be a workaround.  I have mine plugged into a USB port on a monitor.  And for this monitor (I don't know how common this is), when the monitor powers down and goes into its sleep/suspend mode, it cuts the power to its internal USB hub thus turning off the Stream Deck.

**Comment by Core447** [comment]
@pflavio For this you can use the "Go To Sleep" action of the deck plugin - it won't go to sleep automatically but it's better than nothing. Alternatively you can use the screensaver functionality.
